src/ipa_pc/mod.rs

```diff
@@ -14,7 +14,7 @@ pub use data_structures::*;
 use rayon::prelude::*;
 
 use digest::Digest;
-use crate::rng::{FiatShamirRng, FiatShamirChaChaRng};
+use crate::fiat_shamir_rng::{FiatShamirRng, FiatShamirChaChaRng};
```

Comment:
Let us get rid of the comments in line 24-29 below.

Reply:
Done